lib/msal-node/tsconfig.json

```diff
@@ -19,21 +19,25 @@
     "noImplicitThis": true,
     "alwaysStrict": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
```

Comment:
Had to switch to false due to the high number of unused params in `msal-common`.

Reply:
Can we please file an ADO to tackle this later?

Reply:
done